Dockerfile.kube-client-agent.openshift

```diff
@@ -0,0 +1,17 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+
+COPY . /go/src/github.com/coreos/kubecsr
+
+RUN cd /go/src/github.com/coreos/kubecsr \
+  && make bin/kube-client-agent
+
+# stage 2
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+
+ENTRYPOINT ["/usr/bin/kube-client-agent"]
+
+COPY --from=builder /go/src/github.com/coreos/kubecsr/bin/kube-client-agent /usr/bin/
+
+LABEL io.k8s.display-name="kube-client-agent" \
+      io.k8s.description="This client agent generates a valid CSR according to the configuration flags provided to it." \
+      maintainer="Sam Batschelet <sbatsche@redhat.com>"
```

Comment:
Do you need these too? for getting the images into the release
```Dockerfile
io.openshift.tags="openshift" \
io.openshift.release.operator=true \
```

Reply:
yes that is my understanding